editor/src/core/shared/template-path.ts

```diff
@@ -149,6 +149,17 @@ export function toString(target: TemplatePath): string {
   }
 }
 
+export function toStringOptional(target: TemplatePath | null | undefined): string | null {
```

Comment:
unfortunately I couldn't extend toString to be able to return an optional null without having to change too much irrelevant code – so I made a second variant of the function instead

Reply:
I'd argue for using `optionalMap` with the regular `toString` function personally, but this doesn't appear to be being used?

Reply:
I meant to write that I wrote this for debugging convenience, not used in the code. I didn't think about optionalMap, I guess I can just remove this then!

Reply:
removed!